gulpfile.babel.js

```diff
@@ -1,47 +1,52 @@
-var fs = require('fs');
-var path = require('path');
+import fs from 'fs';
+import path from 'path';
 
-var gulp = require('gulp');
+import gulp from 'gulp';
 
 // Load all gulp plugins automatically
 // and attach them to the `plugins` object
-var plugins = require('gulp-load-plugins')();
+import plugins from 'gulp-load-plugins';
 
 // Temporary solution until gulp 4
 // https://github.com/gulpjs/gulp/issues/355
-var runSequence = require('run-sequence');
+import runSequence from 'run-sequence';
 
-var pkg = require('./package.json');
-var dirs = pkg['h5bp-configs'].directories;
+import archiver from 'archiver';
+import glob from 'glob';
+import del from 'del';
+
+import pkg from './package.json';
+
+const dirs = pkg['h5bp-configs'].directories;
 
 // ---------------------------------------------------------------------
 // | Helper tasks                                                      |
 // ---------------------------------------------------------------------
 
-gulp.task('archive:create_archive_dir', function () {
+gulp.task('archive:create_archive_dir', () => {
     fs.mkdirSync(path.resolve(dirs.archive), '0755');
 });
 
-gulp.task('archive:zip', function (done) {
+gulp.task('archive:zip', (done) => {
 
-    var archiveName = path.resolve(dirs.archive, pkg.name + '_v' + pkg.version + '.zip');
-    var archiver = require('archiver')('zip');
-    var files = require('glob').sync('**/*.*', {
+    let archiveName = path.resolve(dirs.archive, `${pkg.name}_v${pkg.version}.zip`);
```

Comment:
Since this variable is never rebound, this [should be](https://mathiasbynens.be/notes/es6-const) `const`, not `let`.